Develop an SQL schema for SOPs

```diff
@@ -1,29 +1,4351 @@
+
+
+-- Completed on 2020-10-30 17:55:35
+
+--
+-- PostgreSQL database dump complete
+--
+
+--
+-- PostgreSQL database dump
+--
+
+-- Dumped from database version 12.3
+-- Dumped by pg_dump version 12.3
+
+-- Started on 2020-10-30 17:54:25
+
+SET statement_timeout = 0;
+SET lock_timeout = 0;
+SET idle_in_transaction_session_timeout = 0;
+SET client_encoding = 'UTF8';
+SET standard_conforming_strings = on;
+SELECT pg_catalog.set_config('search_path', '', false);
+SET check_function_bodies = false;
+SET xmloption = content;
+SET client_min_messages = warning;
+SET row_security = off;
+
+DROP DATABASE "MzTab and TraML";
+--
+-- TOC entry 2965 (class 1262 OID 17145)
+-- Name: MzTab and TraML; Type: DATABASE; Schema: -; Owner: postgres
+--
+
+CREATE DATABASE "MzTab and TraML" WITH TEMPLATE = template0 ENCODING = 'UTF8' LC_COLLATE = 'German_Germany.1252' LC_CTYPE = 'German_Germany.1252';
+
+
+ALTER DATABASE "MzTab and TraML" OWNER TO postgres;
+
+\connect -reuse-previous=on "dbname='MzTab and TraML'"
+
+SET statement_timeout = 0;
+SET lock_timeout = 0;
+SET idle_in_transaction_session_timeout = 0;
+SET client_encoding = 'UTF8';
+SET standard_conforming_strings = on;
+SELECT pg_catalog.set_config('search_path', '', false);
+SET check_function_bodies = false;
+SET xmloption = content;
+SET client_min_messages = warning;
+SET row_security = off;
+
+--
+-- TOC entry 10 (class 2615 OID 17146)
+-- Name: MzTab; Type: SCHEMA; Schema: -; Owner: postgres
+--
+
+CREATE SCHEMA "MzTab";
+
+
+ALTER SCHEMA "MzTab" OWNER TO postgres;
+
+--
+-- TOC entry 4 (class 2615 OID 17171)
+-- Name: MzTab-M; Type: SCHEMA; Schema: -; Owner: postgres
+--
+
+CREATE SCHEMA "MzTab-M";
+
+
+ALTER SCHEMA "MzTab-M" OWNER TO postgres;
+
+--
+-- TOC entry 8 (class 2615 OID 17329)
+-- Name: PQP; Type: SCHEMA; Schema: -; Owner: postgres
+--
+
+CREATE SCHEMA "PQP";
+
+
+ALTER SCHEMA "PQP" OWNER TO postgres;
+
+--
+-- TOC entry 11 (class 2615 OID 17316)
+-- Name: XML; Type: SCHEMA; Schema: -; Owner: postgres
+--
+
+CREATE SCHEMA "XML";
+
+
+ALTER SCHEMA "XML" OWNER TO postgres;
+
+SET default_tablespace = '';
+
+SET default_table_access_method = heap;
+
+--
+-- TOC entry 208 (class 1259 OID 17255)
+-- Name: Metadata_Proteomics; Type: TABLE; Schema: MzTab; Owner: postgres
+--
+
+CREATE TABLE "MzTab"."Metadata_Proteomics" (
+    "mzTab-version" text NOT NULL,
+    "mzTab-mode" text NOT NULL,
+    "mzTab-type" text NOT NULL,
+    "mzTab-ID" text NOT NULL,
+    title text,
+    description text,
+    sample_processing_number text,
+    sample_processing text,
+    instrument_number text,
+    instrument text,
+    "instrument-name" text,
+    "instrument-source" text,
+    analyzer_number text,
+    "instrument-analyzer" text,
+    "instrument-detector" text,
+    software_number text NOT NULL,
+    software text NOT NULL,
+    setting_number text,
+    "software-setting" text,
+    protein_search_engine_score_number text,
```

Comment:
Should the score columns be text or float/double?

Reply:
done